src/preferences/dialog/dlgprefinterfacedlg.ui

```diff
@@ -75,7 +89,65 @@
         </property>
        </widget>
       </item>
+
+      <item row="5" column="1" colspan="2">
+       <widget class="QLabel" name="skinDescriptionText">
+        <property name="sizePolicy">
+         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
+          <horstretch>0</horstretch>
+          <verstretch>0</verstretch>
+         </sizepolicy>
+        </property>
+        <property name="maximumSize">
+         <size>
+          <width>440</width>
+          <height>16777215</height>
+         </size>
+        </property>
+        <property name="text">
+         <string>(skin description goes here)</string>
```

Comment:
Please change to ``<string notr="true">``, to hide the dummy text from the list of our translatable strings.

Reply:
Thanks, done.